### PR TITLE
 WIE-6/ about us cards and aboutWIESoc intro

### PR DESCRIPTION
**Pull request by ammu393**
added to the about us section on web page, with intro and events and programs card

### CODE REVIEW

File: app/(pages)/about-us/page.tsx [hunk 1]
```diff
@@ -1,10 +1,89 @@
...
+
+
+            
   return (
+    <Sheet>
     <Stack>
       <PageHeaderCard imagePath={'/about-us/header.jpg'} pageTitle={'About Us'}/>
     </Stack>
+      <center>
+      <Typography level='h3' fontSize='30px' className='graphite' py={5} sx={{textAlign:'center'}}>
```
[the remark this answers is not in the document]
Comment (angewen) [reply app/(pages)/about-us/page.tsx:57]: You can also use the HeadingBodyText component for this part now 
image

File: app/(pages)/about-us/page.tsx [hunk 2]
```diff
@@ -1,10 +1,89 @@
...
+                          gender acceptance and equality in all facets of the engineering industry.'
+
+
+            
   return (
+    <Sheet>
     <Stack>
       <PageHeaderCard imagePath={'/about-us/header.jpg'} pageTitle={'About Us'}/>
     </Stack>
+      <center>
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:56]: this is a deprecated feature 
image
pls use flex styling on the Stack above it instead, e.g. justifyContent, alignItems

File: app/(pages)/about-us/page.tsx [hunk 3]
```diff
@@ -1,10 +1,89 @@
...
+            src={picture}
+            alt=''
+            />
+          </AspectRatio>
+        </CardOverflow>
+
+        <CardContent>
+          <Stack direction='column' alignItems='center'>
+            <Typography fontSize='30px' pb={3} pt={2} fontWeight='530'
+              sx={{textAlign:'center'}}>
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:27]: textAlign can be outside of sx

File: app/(pages)/about-us/page.tsx [hunk 4]
```diff
@@ -1,10 +1,89 @@
...
+              {cardName}
+            </Typography>
+            <Typography fontSize='16px' px={2} fontWeight='250'
+              sx={{textAlign:'center'}}>
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:31]: textAlign can be outside of sx

File: app/(pages)/about-us/page.tsx (hunk 1 above)
Comment (angewen) [inline app/(pages)/about-us/page.tsx:57]: textAlign can be outside of sx

File: app/(pages)/about-us/page.tsx [hunk 5]
```diff
@@ -1,10 +1,89 @@
...
+      <Typography level='h3' fontSize='30px' className='graphite' py={5} sx={{textAlign:'center'}}>
+          About WIESoc
+        </Typography>
+        <Typography fontSize='20px' pb={10} px={18}>
+          {aboutWIEsocText}
+        </Typography>
+      </center>
+      <Sheet>
+        <Box className='bg-medium-blue' sx={{height: 744, width: '100%'}}>
+        <Typography level='h4' fontSize='30px' className='light-white' pb={3} pt={3} sx={{textAlign: 'center'}}>
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:66]: textAlign can be outside of sx

File: app/(pages)/about-us/page.tsx [hunk 6]
```diff
@@ -1,10 +1,107 @@
...
+          </Box>
+          </Stack>
+        </CardContent>
+      </Card>
+  );
+
+}
 export default function AboutUs() {
+
+  const aboutWIEsocText = 'We are a UNSW student-run society that facilitates members from all \n \
```
Comment (sophiac6410) [inline app/(pages)/about-us/page.tsx:48]: Can you remove the newline \n characters? I'm putting in a style that renders them so it'll look funny haha. You can keep the '\'

**Review by sophiac6410 — APPROVED** [review]

File: app/(pages)/about-us/page.tsx [hunk 7]
```diff
@@ -1,10 +1,107 @@
...
+      </Stack>
+      <HeadingBodyText heading='About WIESoc' body={aboutWIEsocText} color='graphite'/>
+      <Sheet>
+        <Box className='bg-medium-blue' sx={{height: '100%', width: '100%'}}>
+          <Typography level='h1' className='white' textAlign='center' pt={5} >
+              What We Do 
+          </Typography>
+          <Grid container 
+                    justifyContent='flex-end'
+                    sx={{ 'width': {xs: '50%', sm: '100%', md: '100%', lg: '100%', xl: '100%'}, pb:8 }}
```
Comment (angewen) [inline app/(pages)/about-us/page.tsx:68]: you can just have xs and sm

Comment (angewen) [inline app/(pages)/about-us/page.tsx:68]: pb can be outside sx

**Review by angewen — APPROVED** [review]

**Pull request merged**